package.json

```diff
@@ -62,7 +62,7 @@
     "removedrive": "^1.1.1"
   },
   "dependencies": {
-    "angular": "^1.6.3",
+    "angular": "1.6.3",
```

Comment:
The shrinkwrap is already referencing 1.6.3 (given I upgraded the dependency in the other PR), so there's no change there.